SPECS/auoms/auoms.spec

```diff
@@ -0,0 +1,193 @@
+%define      debug_package %{nil}
+
+Summary:        Auditd plugin that forwards audit events to OMS Agent for Linux
+Name:           auoms
+Version:        2.2.5
+Release:        1%{?dist}
+License:        MIT
```

Comment:
Are all of these sources MIT?

Reply:
auoms is MIT, Pal is MIT, msgpack-c is Boost license, RapidJSON is MIT and JSON license.

Reply:
The THIRD_PARTY_IP_NOTICE file listed below includes the MIT license and the Boost license for msgpack-c. The limited use of rapidjson in this package only requires the MIT license.

---
In reply to: [510529327](https://github.com/microsoft/CBL-Mariner/pull/258#discussion_r510529327) [](ancestors = 510529327)